sdk/invoke.js

```diff
@@ -33,24 +34,24 @@ const invoke = async function invoke(accountId, funcName, apiKey, funcData) {
     logger.debug('Invoke response', { statusCode, headers, body });
     return res;
   } catch (err) {
-    let parsedError;
-    try {
-      parsedError = JSON.parse(err.error);
-    } catch (nestedErr) {
-      logger.debug('Failed to parse JSON response', { nestedErr });
-    }
-
-    if (parsedError) {
-      const errorOrMsg = parsedError.error || parsedError.message;
-      if (parsedError.stack && errorOrMsg) {
-        throw new Error(`${errorOrMsg}\n${parsedError.stack}`);
+    const fallbackError = err.error || err;
+    if (err instanceof StatusCodeError) {
```

Comment:
is this important?
what happens if another error is returned?
shouldn't the code try to parse anyhow?